volmdlr/edges.py

```diff
@@ -642,29 +644,33 @@ def __init__(self, degree: int, control_points: List[volmdlr.Point2D],
                                 None, False, name)
 
 
-
 class LineSegment2D(LineSegment):
     """
     Define a line segment limited by two points
     """
 
     def __init__(self, start, end, *, name=''):
+        if isinstance(start, list):
+            start = volmdlr.Point2D(start[0], start[1])
+        if isinstance(end, list):
```

Comment:
Same as previous comment on sequence check : 

```suggestion
        if dc.is_sequence(end):
```